docs/performance.md

```diff
@@ -1,7 +1,5 @@
 Performance
 ===========
 
-Performance Tools
------------------
-
-React offers [Performance Tools](http://facebook.github.io/react/docs/perf.html) that help you get an overview of your app's overall performance. The documentation is fairly complete and should help you get quickly started.
+- React Docs: [Optimizing Performance](https://reactjs.org/docs/optimizing-performance.html)
+- [Debugging React performance with React 16 and Chrome Devtools](https://building.calibreapp.com/debugging-react-performance-with-react-16-and-chrome-devtools-c90698a522ad)
```

Comment:
I'd also link back to the official docs" https://developers.google.com/web/tools/chrome-devtools/rendering-tools/

It feels like Chrome changes things relatively often.

Reply:
Good idea! Updated.